### ISSUE TITLE
Python Binding f.stat core dumping

### ISSUE THREAD

**Issue by efajardo** [xrootd/xrootd#256]
The python bindings fail while doing some stat on some files. Eventhough the files exist they do not fail on open, the stat fails in segmantation fault. Here is the code I used to test it:

Both client and server are 4.2.1. Just ran this script several times against a server and it will output:

Segmentation fault (core dumped)

```python
#!/usr/bin/env python                                                                                                                                         
from XRootD import client
from XRootD.client.flags import OpenFlags
from XRootD.client.responses import XRootDStatus
import random
import sys

xrootd_server = '169.228.130.107'


def main():
   fileNumber = random.choice(range(1,40))
   filename = 'random' + str(fileNumber) + '.root'
   fileLocation = 'root://' + xrootd_server + ':1094//' + '/data/testdata/' + filename
   print 'File to be opened: ' + fileLocation

   with client.File() as f:
      openOut = f.open(fileLocation, OpenFlags.READ)
      status, response = f.stat()
      f.close()
   sys.exit(0);



if __name__ == "__main__":
        main()

```

**Comment by esindril** [comment]
Hi Edgar,

Thanks for the bug report. 
I confirm I can reproduce it and I'll look into it.

Elvin

**Comment by esindril** [comment]
Hi Edgar, 

This issue has already been fixed in the master by the following commit: 98df5250dbfc6f0f8373b51df78a0d2633170e83
The fix will be available in the next release.

Cheers,
Elvin
